examples/batchopenchannel.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+import json
+from pprint import pprint
+import os
+import base64
+import codecs
+from time import sleep
+from datetime import datetime, timedelta
+
+from hashlib import sha256
+import secrets
+
+from yachalk import chalk
+
+
+
+
+
+# Pip installed Modules
+from lndgrpc.client import LNDClient
```

Comment:
you dont necessarily need the .client part here
`from lndgrpc import LNDClient`